sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessSessionMessageEventArgs.cs

```diff
@@ -31,22 +31,19 @@ public class ProcessSessionMessageEventArgs : EventArgs
         public CancellationToken CancellationToken { get; }
 
         /// <summary>
-        /// The <see cref="System.Threading.CancellationToken"/> instance is cancelled when the lock renewal failed to
-        /// renew the lock, or the <see cref="ServiceBusSessionProcessorOptions.MaxAutoLockRenewalDuration"/> has elapsed,
-        /// or when the session lock has been lost, or if <see cref="ReleaseSession"/> is called.
+        /// An event that is raised when the session lock is lost. This event is only raised for the scope of the Process Session Message handler.
+        /// Once the handler returns, the event will not be raised. There are two cases in which this event can be raised:
```

Comment:
We should consider using an actual `<list>` element here so that it renders nice.